src/inmanta/server/services/resourceservice.py

```diff
@@ -544,6 +647,23 @@ async def resource_deploy_done(
                 if status == ResourceState.deployed:
                     extra_fields["last_success"] = resource_action.started
 
+                if status == ResourceState.failed or status == ResourceState.skipped:
```

Comment:
needs to be added to 'resource_action_update' or that one needs to be updated to no longer allow the deploy start/done flow?